src/applications/check-in/components/TravelPayOMB.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+// OMB Details
+const EXP_DATE = '11/30/2027';
+const OMB_NUMBER = '2900-0798';
+const RES_BURDEN_MIN = '10';
+
+const TravelPayOMB = () => {
+  return (
+    <va-omb-info
+      exp-date={EXP_DATE}
+      omb-number={OMB_NUMBER}
+      res-burden={RES_BURDEN_MIN}
+      data-testid="travel-pay-omb"
+      class="vads-u-margin-y--4"
+    >
+      <p>
+        <strong>VA Burden Statement:</strong> An agency may not conduct or
```

Comment:
Does this content need to be translated?

Reply:
I would assume not just on the fact that the component itself doesn't offer any internationalization. But getting confirmation on that.

Reply:
It does require translation. This will be done in a follow up ticket

Reply:
Even though we don't have the translation, this should still get added to the english translation file and reference the translation keys rather than use hard coded text.

Reply:
Done!